operator/v1/types_network.go

```diff
@@ -454,25 +454,56 @@ type NetFlowConfig struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// newCollectors is list of objects featuring a valid IPv{4|6} address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	NewCollectors []NetworkFlowTarget `json:"newCollectors,omitempty"`
 }
 
 type SFlowConfig struct {
 	// sFlowCollectors is list of strings formatted as ip:port with a maximum of ten items
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// newCollectors is list of objects featuring a valid IPv{4|6} address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	NewCollectors []NetworkFlowTarget `json:"newCollectors,omitempty"`
 }
 
 type IPFIXConfig struct {
 	// ipfixCollectors is list of strings formatted as ip:port with a maximum of ten items
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// newCollectors is list of objects featuring a valid IPv{4|6} address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	NewCollectors []NetworkFlowTarget `json:"newCollectors,omitempty"`
 }
 
 // +kubebuilder:validation:Pattern=`^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]):([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$`
 type IPPort string
 
+type NetworkFlowTarget struct {
+	// Address must be a valid ipv4 or ipv6 address.
+	//
+	// +required
+	// +kubebuilder:validation:Required
+	Address string `json:"address,omitempty"`
```

Comment:
There's no reason to put "omitempty" on a field that is required to have a non-empty value. (This applies in a bunch of places...)